client/src/components/molecules/inputs/select-input.tsx

```diff
@@ -14,25 +19,34 @@ const SelectInput: React.FC<SelectInputProps> = ({
     value,
     onChange,
     options,
+    categoryOptionsIsLoading,
     className = '',
     error
 }) => {
     return (
-        <div className={`flex flex-col gap-1 ${className}`}>
-            {label && <label htmlFor={name} className="text-sm font-medium text-foreground">{label}</label>}
+        <div className={`space-y-4 ${className}`}>
```

Comment:
use this instead of string concatenation https://www.npmjs.com/package/tailwind-merge

setup a utility funtction that wraps this module